src/resources/2022/Stillsuit.ts

```diff
@@ -0,0 +1,79 @@
+import { cliExecute, Modifier, toModifier, visitUrl } from "kolmafia";
+import { have as haveItem } from "../../lib.js";
+import { get } from "../../property.js";
+import { $item } from "../../template-string.js";
+
+/**
+ * Do you own a still-suit?
+ * @returns Whether you have the tiny stillsuit
+ */
+export function have(): boolean {
+  return haveItem($item`tiny stillsuit`);
+}
+
+/**
+ * @returns the expected adventures from drinking stillsuit distillate
+ */
+export function distillateAdventures(): number {
+  if (!have()) return 0;
+  const sweat = get("familiarSweat");
+  return Math.round(sweat ^ 0.4);
```

Comment:
[This does not do what you think it does](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR)

instead use ** or Math.pow